<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sc2AiAppDialog</class>
 <widget class="QMainWindow" name="Sc2AiAppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SC2AI Manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="TestTab">
    <attribute name="title">
     <string>Test Local Bot</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>164</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Local Bot Dir</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LocalBotText"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="MapCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Opponent Bot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="RemoteBotCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Testl Bot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="LocalBotCombo"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="LaunchButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseButton">
     <property name="geometry">
      <rect>
       <x>247</x>
       <y>0</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HumanTab">
    <attribute name="title">
     <string>Play Vs Bot</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="HMapCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Opponent Bot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="HRemoteBotCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Race</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="HRaceCombo">
        <item>
         <property name="text">
          <string>Terran</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Zerg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Protoss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="HLocalBotText"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Local Bot Dir</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Player Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="HumanNameText"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="HLaunchButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HBrowseButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
